cli/types/cypress.d.ts

```diff
@@ -59,6 +59,9 @@ declare namespace Cypress {
   interface ObjectLike {
     [key: string]: any
   }
+  interface PromptOptions {
+    excludeFromAI?: Record<string, string | number>
```

Comment:
Thinking about this again, let's just keep it simple for now:
```suggestion
    excludeFromAI?: Record<string, string>
```